### PR TITLE
Refactor command words

### PR DESCRIPTION
**Pull request by nweiyue**

### CODE REVIEW

**Review by erisjacey — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -227,11 +227,11 @@ Students’ data are saved in the hard disk automatically after any command that
 Action | Format, Examples
 --------|------------------
 **Help** | `help`
-**Add** | `add n/NAME m/MATRICULATION_NUMBER e/NUS_EMAIL_ADDRESS [t/TAG]…​` <br> e.g., `add n/Rainer Lam m/A0123456C e/rainerlam@u.nus.edu t/smart`
-**List** | `list`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find Justin Bieber`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX n/UPDATED_NAME`<br> e.g., `edit 2 n/John Cena`
-**Clear** | `clear`
+**Add** | `addstu n/NAME m/MATRICULATION_NUMBER e/NUS_EMAIL_ADDRESS [t/TAG]…​` <br> e.g., `addstu n/Rainer Lam m/A0123456C e/rainerlam@u.nus.edu t/smart`
```
Comment (CodingCookieRookie) [inline docs/UserGuide.md]: Should this be **AddStu** instead of Add? Because there is a AddSes later

**Review by CodingCookieRookie — APPROVED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -10,12 +10,12 @@ ATAS is a **desktop app for managing students’ particulars, optimized for use
...
-  * [Clearing all entries](#clear)
+  * [Adding a students](#addstu)
+  * [Listing all students](#liststu)
+  * [Locating students by name](#findstu)
+  * [Deleting a student](#deletestu)
+  * [Editing a student's particulars](#editstu)
+  * [Clearing all entries](#clearstu)
   * [Switching between tabs](#switch)
   * [Exiting the program](#exit)
   * [User confirmation prompt](#ucp) (coming soon)
```
Comment (marcustw) [inline docs/UserGuide.md]: Perhaps we might want to remove "(coming soon)"?

**Review by marcustw — COMMENTED** [review]
LGTM

File: docs/UserGuide.md (hunk 2 above)
Comment (nweiyue) [reply docs/UserGuide.md]: Good spot!

File: docs/UserGuide.md (hunk 1 above)
Comment (nweiyue) [reply docs/UserGuide.md]: I believe this is to describe the action that users want to perform. I have updated them to "Add Student, Delete Student and so on". Thank you for pointing it out!

**Review by marcustw — APPROVED** [review]
LGTM, bugs fixed!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -9,20 +9,23 @@ ATAS is a **desktop app for managing students’ particulars, optimized for use
...
-  * [Deleting a student](#delete)
-  * [Editing a student's particulars](#edit)
-  * [Clearing all entries](#clear)
+  * [Viewing help](#help)
+  * [Adding a student](#addstu)
+  * [Listing all students](#liststu)
+  * [Locating students by name](#findstu)
+  * [Deleting a student](#deletestu)
+  * [Editing a student's particulars](#editstu)
+  * [Clearing all students](#clearstu)
```
Comment (erisjacey) [inline docs/UserGuide.md]: Maybe this could be changed to `Clearing the student list`?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -327,16 +341,18 @@ Students’ data are saved in the hard disk automatically after any command that
...
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find Justin Bieber`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX n/UPDATED_NAME`<br> e.g., `edit 2 n/John Cena`
-**Addses** | `addses s/SESSION_NAME d/SESSION_DATE`<br> e.g., `addses s/TUT1 d/10/10/2020`
-**Deleteses** | `deleteses INDEX `<br> e.g., `deleteses 2`
-**Editses** | `editses INDEX s/UPDATED_NAME d/UPDATED_DATE`<br> e.g., `editses 2 s/TUT2 d/10/10/2020`
+**Add Student** | `addstu n/NAME m/MATRICULATION_NUMBER e/NUS_EMAIL_ADDRESS [t/TAG]…​` <br> e.g., `addstu n/Rainer Lam m/A0123456C e/rainerlam@u.nus.edu t/smart`
+**List Students** | `liststu`
+**Find Students** | `findstu KEYWORD [MORE_KEYWORDS]`<br> e.g., `findstu Justin Bieber`
+**Delete Students** | `deletestu INDEX`<br> e.g., `deletestu 3`
```
Comment (erisjacey) [inline docs/UserGuide.md]: Maybe this could be changed to `Delete Student`, seeing as we are deleting one student at a time?

**Review by erisjacey — COMMENTED** [review]
LGTM! Just a few nits.

**Pull request merged**